build/scripts/coverage.sh

```diff
@@ -5,8 +5,8 @@
 #
 # Code coverage generation
 CVG_EXCLUDE="${CVG_EXCLUDE:-test}"
-
-go test -coverpkg=./... -coverprofile ./coverage.cov $(go list ./... |  grep -Ev "${CVG_EXCLUDE}")
+go test -coverpkg=./... -coverprofile ./coverageTmp.cov $(go list ./... |  grep -Ev "${CVG_EXCLUDE}")
```

Comment:
Something like this should work.
go test -coverprofile ./coverage.cov \
  $(go list ./... | grep -Ev github.com/verrazzano/verrazzano-operator/test | grep -Ev github.com/verrazzano/verrazzano-operator/pkg/testutil | grep -Ev github.com/verrazzano/verrazzano-operator/pkg/testutilcontroller)